theano/tests/main.py

```diff
@@ -56,6 +51,7 @@ def prepare_test_args(self, verbose=1, extra_argv=None, coverage=False,
 
         Takes the same arguments as `test`.
         """
+        import nose.plugins.builtin
```

Comment:
Add a comment at each of the place you added import not at the top of the file.

Otherwise, in the futur someone will forget why this file isn't pep8 complient and will revert this commit.